panther_driver/src/hardware_node.py

```diff
@@ -46,21 +46,18 @@ def __init__(self) -> None:
         self._watchdog = Watchdog()
         self._watchdog.turn_on()
 
-        rospy.init_node("panther_hardware")
+        rospy.init_node(name, anonymous=False)
 
-        self._e_stop_state_pub = rospy.Publisher(
-            "/panther_hardware/e_stop", Bool, queue_size=1
-        )
-        self._timer_e_stop = rospy.Timer(
-            rospy.Duration(0.1), self._publish_e_stop_state
-        )
+        # -------------------------------
+        #   Publishers
+        # -------------------------------
 
-        self._charger_state_pub = rospy.Publisher(
-            "/panther_hardware/charger_connected", Bool, queue_size=1
-        )
-        self._timer_charger = rospy.Timer(
-            rospy.Duration(0.5), self._publish_charger_state
-        )
+        self._e_stop_state_pub = rospy.Publisher("/panther_hardware/e_stop", Bool, queue_size=1)
+        self._charger_state_pub = rospy.Publisher("/panther_hardware/charger_connected", Bool, queue_size=1)
```

Comment:
```suggestion
        self._e_stop_state_pub = rospy.Publisher("hardware/e_stop", Bool, queue_size=1)
        self._charger_state_pub = rospy.Publisher("hardware/charger_connected", Bool, queue_size=1)
```

We could use the namespace approach

Reply:
This will require changes in webui joystick launch for panther.

Reply:
But those changes are not that drastic to be honest

Reply:
Due to the problems mentioned, we decided to make these changes in the next task.